src/main/java/org/team4909/powerup2018/Robot.java

```diff
@@ -263,3 +260,7 @@ protected void robotDisabled() {
         }
     }
 }
+
+int arduinoElevatorPosition = (30*elevator.getCurrentPosition()/34000);
+
+arduino.send(arduinoElevatorPosition);
```

Comment:
this code can't just hangout here, this isn't JavaScript... you should use it in `robotPeriodic`